attic/README.md

```diff
@@ -245,8 +245,8 @@ other jobs in a simple way.
 
 The basic pattern provided by toil is as follows:
 
-1. You have a job running on your cluster which requires further parallelisation.
-2. You create a list of jobs to perform this parallelisation. These are the 'child' jobs of your process, we call them collectively the 'children'.
+1. You have a job running on your cluster which requires further parallelization.
+2. You create a list of jobs to perform this parallelization. These are the 'child' jobs of your process, we call them collectively the 'children'.
```

Comment:
We can go ahead and change this, but we're likely to find more British-isms around the codebase thanks to our illustrious leader @benedictpaten, which I'd mostly leave alone.